### PR TITLE
pet-image-detection imagestream

### PR DESCRIPTION
**Pull request by isabelizimm**
This PR will address [operatefirst/support#107](https://github.com/operate-first/support/issues/107), by adding the pet-image-detection image so that it is exposed on the JH launcher menu.

### CODE REVIEW

File: odh/base/jupyterhub/notebook-images/pet-image-detection.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    local: true
+  tags:
+    - annotations:
+        openshift.io/imported-from: quay.io/aicoe/pet-image-detection
+      from:
+        kind: DockerImage
+        name: quay.io/aicoe/pet-image-detection:latest
+      importPolicy:
+        scheduled: true
+      name: "latest"
```
Comment (HumairAK) [inline odh/base/jupyterhub/notebook-images/pet-image-detection.yaml:26]: Can you add a newline here?

**Comment by HumairAK** [comment]
@isabelizimm it seems you're failing some linting checks, just a heads up you can run these tests locally by following: https://github.com/operate-first/apps/blob/master/contributing.md#tests

**Comment by isabelizimm** [comment]
> @isabelizimm it seems you're failing some linting checks, just a heads up you can run these tests locally by following: https://github.com/operate-first/apps/blob/master/contributing.md#tests

yes! let me try this locally to see what the issue is

**Comment by HumairAK** [comment]
@isabelizimm when you run `pre-commit run --all-files` it will also attempt to fix the linting issues for you, which can be handy.

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by 4n4nd — APPROVED** [review]

**Pull request merged**